rand_chacha/src/chacha.rs

```diff
@@ -21,52 +20,6 @@ const BUF_BLOCKS: u8 = 4;
 // number of 32-bit words per ChaCha block (fixed by algorithm definition)
 const BLOCK_WORDS: u8 = 16;
 
-#[repr(transparent)]
-pub struct Array64<T>([T; 64]);
```

Comment:
This was needed because of the bound `Results: Default` and `[_; 64]` not implementing this.

It's a nice but ultimately unimportant improvement of the new utility fns.